.github/PULL_REQUEST_TEMPLATE/manual_maintenance.md

```diff
@@ -11,46 +11,49 @@ creation URL (or simply copy this md file into the description)
 
 - See 'Problematic Updates' in the wiki (evaluate if issues persist or if newer versions are available that fix them)
 
-- [ ] Take a look at the dependency dashboard and see which updates are to be applied. For a detailed description of the process,
-consult the internal Dataland Wiki.
+- [ ] Take a look at the dependency dashboard and see which updates are to be applied. Never press the "Create all rate-limited PRs at once" box. For a detailed description of the process,
+  consult the internal Dataland Wiki.
 
 - [ ] Update the wiki page (should display current issues why specific updates were not done)
 
-### Dataland EuroDaT client
+### Dataland EuroDaT client (The EuroDaT section should be skipped)
 
 - [ ] Check on the https://eurodat.gitlab.io/trustee-platform/release_notes/ if there is a newer version available, if yes
   then update the version number used in docker-compose.
-- [ ] If the version was changed: Check if the eurodatClientOpenApi.json in dataland-external-storage is in sync with 
-  the currently used version of the client. 
+- [ ] If the version was changed: Check if the eurodatClientOpenApi.json in dataland-external-storage is in sync with
+  the currently used version of the client.
   1. Edit the docker compose file by adding a `ports` section with the entry `"8080:8080"` to the "eurodat-client" and start it
-  2. Open a shell and navigate to the `dataland-eurodat-client` subproject 
+  2. Open a shell and navigate to the `dataland-eurodat-client` subproject
   3. Execute the following snippet of code (requires python): `curl http://localhost:8080/api/v1/client-controller/openapi | sed 's/\(example: \)\([^ ]*\)/\1"\2"/g' | python -c 'import sys, yaml, json; print(json.dumps(yaml.safe_load(sys.stdin.read()), indent=2))' > ./eurodatClientOpenApi.json`
   4. If there are changes to `eurodatClientOpenApi.json`, discuss with the team how to proceed
 
 ## Server maintenance
 
-Note: Before applying any update to any server make sure that a backup exists. In case of prod, create a fresh backup just
+Note: Before applying any update to any server make sure that one backup exists (One backup for the following dev servers, letsencrypt, clone and test together is enough). In case of prod, create a fresh backup just
 before applying any changes and align with the team when to apply them.
 
+For creating a backup please refer to: [How to create a backup on OTC](https://dfinegmbh.sharepoint.com/:v:/r/sites/TMDataland/Freigegebene%20Dokumente/General/OTC%20Documentation/BackupAndRestore.mkv?csf=1&web=1&e=blez3)
```

Comment:
Den link nicht hier in der public code base. Verweis auf die Anleitung im internen Wiki reicht.